src/server/saved_objects/service/create_saved_objects_service.js

```diff
@@ -82,6 +82,13 @@ export function createSavedObjectsService(server) {
     }
   });
 
+  const unscopedClientProvider = {
+    getClient(callCluster) {
```

Comment:
I hope I'm understanding well enough to do what I should be doing here. The scopedClientProvider is doing a lot more in its `getClient` method. I did as much as I needed to get the code to work :D